apps/admin_app/lib/admin_app_web/templates/order/address_add.html.eex

```diff
@@ -0,0 +1,84 @@
+<div class="container">
+  <nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+      <li class="breadcrumb-item"><a href="/orders">Orders</a></li>
+      <li class="breadcrumb-item active" aria-current="page"><%= @order.number %></li>
+    </ol>
+  </nav>
+
+  <h2>
+    Order #<%= @order.id %>
+    <%= state_badge(@order.state) %>
+  </h2>
+  <div class="row">
+    <div class="col-8">
+      <!-- The real deal  -->
+      <div class="card">
+        <h3>Add Address </h3>
+        <div class="card-body">
+          <form action=<%= "/orders/#{@order.number}/address" %> method="post">
```

Comment:
Use the form structure as shown in `https://github.com/aviabird/snitch/blob/develop/apps/admin_app/lib/admin_app_web/templates/stock_location/_form.html.eex`